catalogs/bodleian.yaml

```diff
@@ -1,7 +1,7 @@
 metadata:
   version: 1
   data_path: bodleian
-  schedule: "40 8 * * *"
+  schedule: "30 11 01 Jan,Apr,Jul,Oct"
```

Comment:
This is supposed to be 11:30am (assuming this is Pacific time) the 1st of Jan, Apr, Jul, Oct

Reply:
@jacobthill you left off the "day of the week" notation, so this should be:

```
30 11 01 Jan,Apr,Jul,Oct *
```

As well as the other ones. That will fix the error on startup.